DbgPack/__init__.py

```diff
@@ -1,7 +1,11 @@
-from .asset import Asset
+from .asset1 import Asset1
+from .asset2 import Asset2
+from .loose_asset import LooseAsset
+from .asset_manager import AssetManager
 from .pack1 import Pack1
 from .pack2 import Pack2
+from .loose_pack import LoosePack
 from .struct_reader import BinaryStructReader
```

Comment:
`BinaryStructReader` is an internal utility class, it doesn't make a lot of sense to expose it. It should probably be renamed to have a `_` prefix to communicate this.
```suggestion
```